blackmarket/UI/Common/StateButton.swift

```diff
@@ -13,7 +13,7 @@ struct StateButton: View {
     let title: String
     @State private var isEnabled: Bool = false
     @State private var isLoading: Bool = false
-    let isValid: AnyPublisher<Bool, Never>
+    var isValid: AnyPublisher<Bool, Never> = Just(true).eraseToAnyPublisher()
```

Comment:
I think this should be a let constant, and the default value should be provided during initialization:
`init(isFetching:.., isValid: AnyPublisher<Bool, Never> = Just(true).eraseToAnyPublisher())`

Reply:
Do you mean something like this? 

`StateButton(action: {}, title: "Button", isValid: Just(true).eraseToAnyPublisher(), isFetching: isFetching)`